main.py

```diff
@@ -1,386 +0,0 @@
-""" generate metabolite data in latent variable representation """
```

Comment:
Did you move this file somewhere?

Reply:
Yep -- it doesn't belong in the refactored structure. The current PR is really just a placeholder -- it shouldn't be merged until all tests are passing, which means the refactor needs to be finished and modules need to be working right.